qt/aqt/main.py

```diff
@@ -284,6 +284,11 @@ def setupProfile(self) -> None:
             profs = self.pm.profiles()
             if len(profs) == 1:
                 self.pm.load(profs[0])
+            elif self.pm.load_last_loaded_profile() and (
+                name := self.pm.last_loaded_profile_name()
+            ):
+                self.pm.load(name)
```

Comment:
What happens when the user deletes the last-loaded profile then starts Anki again?

Reply:
Thanks for spotting that, should be fixed in the latest commit.

Btw, is there any reason that the type hint for `profiles()` in `aqt/profiles.py` is:
`def profiles(self) -> list:` rather than `def profiles(self) -> list[str]:` ?

Reply:
Some of the typings were added using an automatic tool that did not create perfect types